client/account-recovery/forgot-username/forgot-username-form/index.jsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import FormLabel from 'components/forms/form-label';
+import FormInput from 'components/forms/form-text-input';
+
+export class ForgotUsernameFormComponent extends Component {
+	constructor() {
+		super( ...arguments );
```

Comment:
There are a few [weird quirks](https://javascriptweblog.wordpress.com/2011/01/18/javascripts-arguments-object-and-beyond/) with the `arguments` object, a spread operator can be used to achieve the same thing:
```js
constructor( ...args ) {
	super( ...args );
}
```